staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/convert.go

```diff
@@ -68,11 +68,8 @@ func NewStructural(s *apiextensions.JSONSchemaProps) (*Structural, error) {
 		ValueValidation: vv,
 	}
 
-	if s.Items != nil {
-		if len(s.Items.JSONSchemas) > 0 {
```

Comment:
can we keep this check and error, and just make the assignment to ss.Items conditional in s.Items.Schema being non-nil? depending on validation done elsewhere seems fragile

Reply:
we have the problem that we do not output proper validation errors if this already fails.

Note, either JSONSchemas or Schema is set. It's a union.

Reply:
ok